scripts/prepare-machine.ps1

```diff
@@ -137,12 +140,13 @@ function Download-CoreNet-Deps {
 }
 
 function Download-Xdp-Kit {
+    if ($Force) { rm -Force -Recurse $ArtifactsPath }
```

Comment:
Can we make the -Force behavior the regular behavior?

Reply:
I'd prefer not, since it's not that way for any/all other PS commands/scripts.

Reply:
In what situation would you not want to pass -Force?

Reply:
It's just the general pattern to not overwrite things unless an explicit signal is given to do so. I've been considering rewriting this entire scripts BTW, and perhaps I can figure out a better way to do this then.